### PR TITLE
[fix](insert-into) fix insert into lose data

### PR DESCRIPTION
**Pull request by sollhui**
## Proposed changes

There are some periodic reports during the load process, and the reports from the intermediate process may be concurrent with the last report. The last report causes the counter to decrease to zero, but it is possible that the report without commit-info triggered the commit operation, resulting in the data not being published.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java [hunk 1]
```diff
@@ -2520,7 +2520,7 @@ public void updateFragmentExecStatus(TReportExecStatusParams params) {
                         params.getBackendId(), status.getErrorMsg());
                 updateStatus(status, params.getFragmentInstanceId());
             }
-            if (ctx.fragmentInstancesMap.get(params.fragment_instance_id).getIsDone()) {
```
Comment (liaoxin01) [inline fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java:2523]: Please add a comment.

**Comment by liaoxin01** [comment]
Please add a description of the PR.

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**